ecosystem/sep-0031.md

```diff
@@ -41,7 +41,13 @@ The JWT should be included as a request header:
 Authorization: Bearer <JWT>
 ```
 
-Any API request the fails to meet proper authentication should return a 401 Unauthorized response.
+Any API request the fails to meet proper authentication should return a 403 Forbidden response with the following body
```

Comment:
We _do not_ need this error in the response body. SEP-6 does this because it uses 403 for other response types other than authentication error responses.

Reply:
We can just change it to 403 without a body

Reply:
updated